layouts/joomla/form/field/url.php

```diff
@@ -69,5 +69,5 @@
 echo $name; ?>" <?php
 echo !empty($class) ? ' class="' . $class . '"' : ''; ?> id="<?php
 echo $id; ?>" value="<?php
-echo htmlspecialchars(JStringPunycode::urlToUTF8($value), ENT_COMPAT, 'UTF-8'); ?>" <?php
+echo htmlspecialchars((string) JStringPunycode::urlToUTF8($value), ENT_COMPAT, 'UTF-8'); ?>" <?php
```

Comment:
```suggestion
echo htmlspecialchars((string) JStringPunycode::urlToUTF8($value), ENT_COMPAT, 'UTF-8'); ?>" <?php
```

What happend here that string punycode returns something else than a string? If anything this should be fixed within that methond